paystack/build.gradle

```diff
@@ -74,12 +75,13 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$versions.coroutines"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$versions.coroutines"
 
+    implementation 'com.pusher:pusher-java-client:2.2.1'
```

Comment:
For the sake of consistency, we should follow the established pattern of `$versions.pusherClient`

Reply:
Done @nathanstasin